src/components/v5/shared/SearchSelect/partials/SearchItem/SearchItem.tsx

```diff
@@ -71,13 +96,26 @@ const SearchItem: FC<SearchItemProps> = ({
                     'text-gray-400 pointer-events-none gap-1': isDisabled,
 
                     'justify-center': !isLabelVisible,
+                    'cursor-pointer': !!checkboxesList,
                   },
                 )}
+                name={value.toString()}
                 onClick={() => {
                   onChange?.(value);
                 }}
+                withoutButton={!!checkboxesList}
               >
                 <div className="relative w-full flex items-center">
+                  {checkboxesList && (
```

Comment:
hmmm looking at all the conditional rendering in this file makes me think if it would be beneficial to create a `CheckboxSearchItem` or something along those lines, so we can render that from `SearchSelect` and leave this component intact.